src/utils/dom.js

```diff
@@ -50,7 +50,7 @@ export const eventOff = (el, evtName, handler, options) => {
 
 // Determine if an element is an HTML Element
 export const isElement = el => {
-  return el && el.nodeType === Node.ELEMENT_NODE
+  return Boolean(el && el.nodeType === Node.ELEMENT_NODE)
```

Comment:
Why do we need to wrap this into a `Boolean()`?

Reply:
Needed because we need a true `true` and `false` value.  otherwise, if `el` is not truthy, it returns the el value (i.e. null or undefined)

Reply:
Made the tests a bit annoying when it didn't return a boolean.